doc/user/content/sql/functions/now_and_mz_logical_timestamp.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "now and mz_logical_timestamp functions"
+description: "In Materialize mz_logical_timestamp() acts the way now() does in most systems."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+In Materialize, `now()` doesn't represent the system time; it represents the time with timezone when the query was executed. By contrast, `mz_logical_timestamp()` represents the logical time at which the query executes.
+
+In most Materialize queries, you should use `mz_logical_timestamp()` where you might use `now()` in other systems, for example, when defining [temporal filters](https://materialize.com/temporal-filters/) or when creating materialized views from multiple sources with potentially conflicting system times.
```

Comment:
> or when creating materialized views from multiple sources

I don't think we allow `mz_logical_timestamp` in views unless it's in a temporal filter? If that's correct, this phrase should maybe be removed.